rest/client.go

```diff
@@ -286,3 +298,67 @@ func (c *client) GetOriginNodeID(ctx context.Context, origin models.Origin) (uui
 
 	return response.NodeID, nil
 }
+
+func (c *client) GetAssetComponent(ctx context.Context, assetID, componentID uuid.UUID) (response models.WebmodelsComponent, err error) {
+	request := rest.Get("assets/{node}/components/{component}").
+		SetHeader("Accept", "application/json").
+		Assign("node", assetID).
+		Assign("component", componentID)
+
+	err = c.DoAndUnmarshal(ctx, request, &response)
+
+	return
+}
+
+func (c *client) GetAssetComponents(ctx context.Context, assetID uuid.UUID, filter ComponentsFilter) (response models.WebmodelsComponents, err error) {
+	if filter.Limit == 0 {
+		filter.Limit = 25
+	}
```

Comment:
Long time since I did anything with Asset Components, should we control this limit within the client or let the server set the default?

Reply:
Looks like it's handled on the server end. I'll remove this.